plugins/node.d/smart_

```diff
@@ -589,6 +589,23 @@ def find_smart_drives():
                     continue
         except Exception as exc:
             verboselog('Failed to list FreeBSD disks: {}'.format(exc))
+    elif os.uname()[0] == "MidnightBSD":
+        try:
+            sysctl_kerndisks = os.popen('sysctl kern.disks')
+            kerndisks = sysctl_kerndisks.readline().strip()
+            for drive in kerndisks.split()[1:]:
+                if drive[:2] in ['md', 'cd', 'fd']:
+                    # Ignore Memory Disks, CD-ROM drives and Floppy
+                    continue
+                try:
+                    verboselog('Trying '+drive+'...')
```

Comment:
Please add spaces around operators (`+`). [PEP8](https://www.python.org/dev/peps/pep-0008/) (and `flake8`) would appreciate this.